draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -621,8 +621,16 @@ Other specifications or profiles of this specification may define alternative fo
 
 Security considerations in this section help achieve the following properties:
 
-* Claim values issued by the Issuer cannot be obtained by the adversary, if the Holder did not intend to disclose them.
-* Claim values that are not issued by the Issuer cannot be successfully verified by the Verifier, if the Holder forged the Disclosures.
+**Selective Disclosure:** An adversary in the role of the Verifier cannot obtain
```

Comment:
```suggestion
* An adversary in the role of the Verifier cannot obtain
```
I find two abstractions of a property achieved by security considerations confusing.

Reply:
Not sure what you mean by abstractions, but it is usually helpful to introduce names for the security properties you want to achieve. I'm open to suggestions for better names.

Reply:
is "selective disclosure" a security property..?

Reply:
It is if we define it, and I think we should because that's the main goal of the spec. I'm open to better names, though.